include/seastar/core/gate.hh

```diff
@@ -61,21 +63,16 @@ class gate {
     std::optional<promise<>> _stopped;
 
 #ifdef SEASTAR_GATE_HOLDER_DEBUG
-    size_t _holders = 0;
-
-    void assert_not_held_when_moved() const noexcept {
-        assert(!_holders && "gate moved with outstanding holders");
-    }
-    void assert_not_held_when_destroyed() const noexcept {
-        assert(!_holders && "gate destroyed with outstanding holders");
-    }
+        void assert_not_held_when_moved() const noexcept;
```

Comment:
Why did indentation shifted one level right?

Reply:
not sure. Will fix.

Reply:
fixed in v2